Don't throw errors when starting the backend if no tuner was defined.  The user can configure it later.
http://code.mythtv.org/trac/ticket/9626

Index: mythtv-master/mythtv/programs/mythbackend/scheduler.cpp
===================================================================
--- mythtv-master.orig/mythtv/programs/mythbackend/scheduler.cpp	2011-08-10 07:28:51.354337509 +0800
+++ mythtv-master/mythtv/programs/mythbackend/scheduler.cpp	2011-08-10 07:29:05.306337517 +0800
@@ -172,8 +172,7 @@
         LOG(VB_GENERAL, LOG_ERR, LOC +
                 "No capture cards are defined in the database.\n\t\t\t"
                 "Perhaps you should re-read the installation instructions?");
-        error = GENERIC_EXIT_SETUP_ERROR;
-        return false;
+        return true;
     }
 
     query.prepare("SELECT sourceid,name FROM videosource ORDER BY sourceid;");
